.github/actions/build_portBLAS_action/action.yml

```diff
@@ -17,12 +17,13 @@ runs:
     - name: Set up Environment and Build portBLAS
       shell: bash
       run: |
-        ls
-        export LD_LIBRARY_PATH=${{ inputs.workspace }}/linux_nightly_release/lib:$LD_LIBRARY_PATH
-        export CMAKE_CXX_COMPILER=${{ inputs.workspace }}/linux_nightly_release/bin/clang++
-        export CMAKE_C_COMPILER=${{ inputs.workspace }}/linux_nightly_release/bin/clang
+        export LD_LIBRARY_PATH=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/lib/libsycl.so:${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/lib:$LD_LIBRARY_PATH
```

Comment:
Does setting `LD_LIBRARY_PATH` to a specific `.so` actually do anything? I've have thought that pointing it to `/lib` as you're also doing would be enough.

Reply:
I think you are right, it's just a path.